.github/workflows/Pkg EdFi.Ods.Minimal.Template.PostgreSQL.yml

```diff
@@ -96,6 +96,15 @@ jobs:
       shell: pwsh
       run: |
         .\build.githubactions.ps1 CheckoutBranch -RelativeRepoPath "../Ed-Fi-ODS-Implementation"
+    - name: Install postgresql-client-16 package
```

Comment:
This step will be unnecessary once ubuntu-latest points to ubuntu-24.04
24.04 uses PostgreSQL 16, so there will be no need to reinstall this client version